### PR TITLE
Hotfix bad encoding and bad taskGroudId

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Needs a follow up to clean up the fixes.

### CODE REVIEW

File: src/graph2tasks/cli.py [hunk 1]
```diff
@@ -62,9 +62,11 @@ def main(graph, start, task_group_id, taskcluster_config, dry_run):
 
     tasks = inject_dummy_tasks(tasks, dummy_task)
     tasks = add_atomic_task(tasks, (atomic_task_id, atomic_task))
-    tasks = inject_task_group_id(tasks, task_group_id)
+    tasks = inject_task_group_id(tasks, atomic_task_id)
```
Comment (rail) [inline src/graph2tasks/cli.py]: Can you kill the `--task-group-id` option too. It doesn't look we are using it.

Comment (JohanLorenzo) [reply src/graph2tasks/cli.py]: Done

Comment (rail) [reply src/graph2tasks/cli.py]: Sorry for torturing you, but it's still has to be removed from the function signature of `main` and `_main`

Comment (JohanLorenzo) [reply src/graph2tasks/cli.py]: You're right. I cleaned that up. Thanks for pointing it out !

**Pull request merged**